src/mechanics/timeIntegration/postProcessing/PostProcessor.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <string>
+#include <boost/ptr_container/ptr_vector.hpp>
+
+#include "mechanics/structures/StructureBase.h"
+#include "mechanics/timeIntegration/TimeControl.h"
+#include "mechanics/timeIntegration/postProcessing/ResultBase.h"
```

Comment:
All those includes can be forwarded, right?
(StructureBase, TimeControl, ResultBase)